src/components/searchWrapper/searchForm/chipItem/index.js

```diff
@@ -0,0 +1,41 @@
+import { Component } from 'substance';
+
+class ChipItem extends Component {
+  /**
+   * Render the ChipItem.
+   *
+   * @param  {function} $$ The createComponent function.
+   * @return {VirtualComponent} The ChipItem component.
+   */
+  render($$) {
+    const {
+      keyword,
+    } = this.props;
+    const textId = keyword.replace(/\s/g, '-');
```

Comment:
Suggestion: I always feel that destructuring is better when there are a lot of fields. In this case, I would not. But I understand that's maybe just a preference. So feel free to disregard this suggestion. :)

```suggestion
    const textId = this. props.keyword.replace(/\s/g, '-');
```